lib/index.js

```diff
@@ -15,6 +15,7 @@ class Oppsy extends Events.EventEmitter {
 
         super();
         config = config || {};
+        this.config = config || {};
```

Comment:
I don't like attaching `config` to `this` because it increases the surface area of the object. However, since there isn't a way to pass the config into `stop`, I guess it's ok. Change it to `this._config` though.